examples/llama2-7b/base-model.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: substratus.ai/v1
+kind: Model
+metadata:
+  name: llama-2-7b
+spec:
+  image: substratusai/model-loader-huggingface
+  params:
+    name: meta-llama/Llama-2-7b-hf
+    hugging_face_hub_token: ${HUGGINGFACE_TOKEN}
```

Comment:
Its the user supposed to replace this before applying? If so, we should have a comment here.

Reply:
yes I was using this:
```
export HUGGINGFACE_TOKEN=mahtoken
cat examples/llama2-70b/base-model.yaml | envsubst | kubectl apply -f -
```

I will make a comment in the file itself and write these steps in user doc

Reply:
I added a comment, please let me know if that's good enough.

Reply:
suggestino added above: https://github.com/substratusai/substratus/pull/146/files#r1285113274